test/integration/targets/one_host/tasks/main.yml

```diff
@@ -0,0 +1,221 @@
+# test code for the one_host module
```

Comment:
Maybe you could add a few tests for check mode (`check_mode: yes`) so that we know that the module doesn't change anything in check mode. I remember some Ansible modules having bugs and changing things in check mode.

Reply:
one_host does not support check mode.
I want to add check mode at a later stage.